### PR TITLE
ISSUE 3834 - Upgrade to node 18

### LINKED ISSUE #3834: Update nodejs to 18.12.1 LTS
### Description
Update nodejs to latest [18.12.1 LTS](https://nodejs.org/dist/v18.12.1/node-v18.12.1-linux-x64.tar.xz)

### Tasks
- [x] Update .io
- [x] Update GitHub Actions
- [x] Update Travis CI
- [x] Update containers https://github.com/3drepo/DevOps/issues/504
- [x] Update local https://github.com/3drepo/local/issues/64

### PR DESCRIPTION
**Pull request by Charence**
This fixes #3834 

#### Description
Upgrade from node 14 to node 18 (latest LTS)

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -55,15 +55,15 @@
...
+    "mongodb": "3.7.3",
     "multer": "1.4.1",
     "nodemailer": "6.6.3",
     "openapi-schema-validator": "12.0.0",
     "paypal-rest-sdk": "1.8.1",
     "pug": "3.0.2",
     "serialize-javascript": "4.0.0",
     "serve-favicon": "2.5.0",
-    "sharp": "0.30.3",
+    "sharp": "^0.31.3",
```
Comment (carmenfan) [inline backend/package.json:66]:
```suggestion
    "sharp": "0.31.3",
```

File: backend/tests/v5/helper/services.js [hunk 2]
```diff
@@ -206,7 +206,7 @@ ServiceHelper.generateRandomDate = (start = new Date(2018, 1, 1), end = new Date
     + Math.random() * (end.getTime() - start.getTime()));
 ServiceHelper.generateRandomNumber = (min = -1000, max = 1000) => Math.random() * (max - min) + min;
 
-ServiceHelper.generateRandomURL = () => `http://${ServiceHelper.generateRandomString()}.${ServiceHelper.generateRandomString(3)}/`;
+ServiceHelper.generateRandomURL = () => `http://${ServiceHelper.generateRandomString()}.com`;
```
Comment (carmenfan) [inline backend/tests/v5/helper/services.js:209]: why not keep the / so you don't have to change everything else?

**Review by carmenfan — CHANGES_REQUESTED** [review]

**Comment by carmenfan** [comment]
@Charence  I'm guessing you need https://github.com/3drepo/DevOps/issues/504 https://github.com/3drepo/local/issues/64 merged in or it'll break anyone who's using the containerised build?

Can you work it out with @keifgwinn

**Comment by keifgwinn** [comment]
The dockerfile needs update also

**Review by keifgwinn — APPROVED** [review]

**Comment by keifgwinn** [comment]
approved

**Pull request merged**